torch/jit/_fuser.py

```diff
@@ -48,6 +50,12 @@ def fuser(name):
         torch._C._jit_override_can_fuse_on_gpu(False)
         torch._C._jit_set_texpr_fuser_enabled(False)
         torch._C._jit_set_nvfuser_enabled(True)
+    elif name == 'fuser3': # oneDNN Graph
+        torch._C._jit_override_can_fuse_on_cpu(old_cpu_fuse)
```

Comment:
May I know why we need to set the state to `old_cpu_fuse` here?

Reply:
I was not sure about this and chose to keep the previous setting earlier. Same for your comment below. 
Are you suggesting to set both `True` like nnc?

Reply:
Oh I was just asking since I haven't seen such settings in 'fuser0' `fuser1` or `fuser2` to keep the old status.
I think it should be fine to just set the old status back in the `finally` statement.